### ISSUE TITLE
Building MacOSX

### ISSUE THREAD

**Issue by martynbristow** [astropy/astropy#2011]
I'm trying to build Astropy V0.3 on MacOSX Maverics after finding PIP wouldn't work.
I get:
unable to execute gcc-4.2: No such file or directory
error: command 'gcc-4.2' failed with exit status 1

Basically the same error as through PIP

If i just use python setup.py install i get

The C compiler used to compile Python gcc-4.2, and
which is normally used to compile C extensions, is not
available. You can explicitly specify which compiler to
use by setting the CC environment variable, for example:

```
CC=gcc python setup.py <command>
```

or if you are using MacOS X, you can try:

```
CC=clang python setup.py <command>
```

Given its trying to use gcc-4.2 is it set somewhere in the package

I found a solution...
Basically python tries to use gcc-4.2 which isn't within xtools
Although you may have it somehow.
Run:
sudo ln -s /usr/bin/gcc /usr/bin/gcc-4.2
then it will work

**Comment by bsmith18** [comment]
Why not update the astropy installer to use a modern compiler rather than gcc-4.2? My workaround was linking gcc-4.2 to clang `sudo ln -s /usr/bin/clang /usr/bin/gcc-4.2`
